es.js

```diff
@@ -381,8 +381,8 @@ settings.viewmodel.weapReloadY Recarga de Arma en Y
 settings.viewmodel.yOffADS Ubicación de arma al Apuntar en Y
 settings.viewmodel.adsFovMlt Intensidad del campo de visión al Apuntar
 settings.viewmodel.toggleADS Alternar mira en primera persona
-settings.viewmodel.wallLean Wall Slide/Jump Lean
-settings.viewmodel.wallLeanMlt Wall Slide/Jump Lean
```

Comment:
`Deslización` doesn't exists, change it to: `Deslizarse sobre pared/Salto apoyado` would be better.